### PR TITLE
bump deps

### PR DESCRIPTION
**Pull request by thehesiod**
superset of https://github.com/aio-libs/aiobotocore/pull/735

### CODE REVIEW

**Comment by thehesiod** [comment]
also fixes https://github.com/aio-libs/aiobotocore/issues/695

**Comment by thehesiod** [comment]
before deploying this I need to know why https://github.com/aio-libs/aiobotocore/pull/715 was done, as `aws-xray-sdk` just has the dep: `botocore>=1.11.3`

**Comment by thehesiod** [comment]
ok @bbrendon I think this is ready, just need to hear back from https://github.com/aio-libs/aiobotocore/pull/715 before merging this back in

**Comment by bbrendon** [comment]
Hah. By “kicking the can” I have a feeling the home assistant deps are going to need more changes but that’s fine. I’m more in favor of getting it resolved!

Once it’s on pypi I’ll dig in. Thanks for your work on this.

File: setup.py [hunk 1]
```diff
@@ -25,9 +25,11 @@ def read(f):
...
 
 extras_require = {
-    'awscli': ['awscli==1.16.199'],
-    'aws-xray-sdk': ['aws-xray-sdk==2.4.2'],
-    'boto3': ['boto3==1.9.189'],
+    'awscli': ['awscli==1.16.262'],
+    'boto3': ['boto3==1.9.252'],
+
+    # This is due to a version incompatibility
+    'aws-xray-sdk': ['aws-xray-sdk>=2.4.2'],
```
Comment (oba11) [inline setup.py]: 👍

**Comment by albertferras** [comment]
What's pending to merge this? I am very looking forward for this to be merged and released, `botocore`

**Comment by thehesiod** [comment]
I'm worried about adding the `aws-xray-sdk` extra as I don't think it should go there given it's open version and always has been , I needed to run some tests to see what the issue was.  Perhaps I can delay  that particular part of this PR.

**Pull request merged**